src/scripts/main.js

```diff
@@ -1,3 +1,268 @@
 'use strict';
 
-// write code here
+// notification
+const pushNotification = (posTop, posRight, title, type) => {
+  const notification = document.createElement('div');
+  const h2 = document.createElement('h2');
+
+  h2.innerText = title;
+  h2.classList.add('title');
+
+  notification.classList.add('notification', `${type}`);
+  notification.setAttribute('data-qa', 'notification');
+  notification.appendChild(h2);
+
+  const page = document.body;
+
+  page.insertAdjacentElement('afterbegin', notification);
+  notification.style.top = `${posTop}px`;
+  notification.style.right = `${posRight}px`;
+
+  setTimeout(() => {
+    notification.style.display = 'none';
+  }, 2000);
+};
+
+// sorting
+let currentSortColumn = -1;
+let currentSortOrder = 'ASC';
+
+function sortBy(sortByWhat, sortCount) {
```

Comment:
```suggestion
function sort(sortBy, columnIndex) {
```
or `function sortEmployees`

These names would be a bit better)